descriptor/fingerprint/src/main/java/org/openscience/cdk/fingerprint/AtomPairs2DFingerprinter.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Note: I adapted this fingerprint from Yap Chun Wei's PaDEL source code, which can be found here:
+ * http://www.yapcwsoft.com/dd/padeldescriptor/
+ * 
+ */
```

Comment:
This needs a clear copyright/license statement. See many other source code files, but it must include:

- who wrote the code (so, that includes at least Lyle and Yap)
- the original code's license applies, but since that is public domain (PaDEL website: "The source code is released as public domain."), you actually in this rare occasion the right to set a license (but PD is fine)